synapse/replication/tcp/resource.py

```diff
@@ -38,7 +38,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ReplicationStreamProtocolFactory(Factory):
+class ReplicationStreamProtocolFactory(ServerFactory):
```

Comment:
What's the motivation here?

Reply:
Ahh, [this](https://github.com/twisted/twisted/blob/5c24e99e671c4082a1ddc8dbeb869402294bd0dc/src/twisted/internet/protocol.py#L483) explains it. Nice spot!